### DISCUSSION TITLE
Query on Inverting Plots in Foxglove

### DISCUSSION THREAD

**Discussion by shannonwarren** [foxglove/community#1051] [General]
I’m working with an underwater robot where depth is considered positive underwater. To better align with our application, I’d like to invert ,y depth plot on Foxglove so that positive depth values are displayed in the negative y-direction on the graph.

Is there a way to achieve this inversion directly in Foxglove?

**Comment by shannonwarren** [comment]
this is what I'm looking for using series plots in plotjuggler

![Screenshot from 2024-10-30 18-24-57](https://github.com/user-attachments/assets/341ad01c-fc0d-4c2a-a754-94c66ee5170f)

**Reply by jtbandes** [reply]
You have two main options for plotting modified/derived values:

- Plot panel supports a handful of [math modifiers](https://docs.foxglove.dev/docs/visualization/panels/plot/#math-modifiers) including `.@negative`
- Use a [user script](https://docs.foxglove.dev/docs/visualization/panels/user-scripts/) and use any JavaScript code to produce the value you want to plot

There is currently no option to invert the direction of the axis itself.

**Reply by shannonwarren** [reply]
Perfect. Let me try the math modifiers and get back to you.

**Reply by shannonwarren** [reply]
Thanks @negative kinda works for me for now.
